homeassistant/components/generic/icons.json

```diff
@@ -0,0 +1,5 @@
+{
+  "services": {
+    "reload": "mdi:refresh"
```

Comment:
```suggestion
    "reload": "mdi:reload"
```

Reply:
The chosen icon looks ok to me, but when I search the rest of the repository, the universal choice for icons.json:reload is `mdi:reload` so I suggest using that rather than something different.